### ISSUE TITLE
Toggling between ETH and USD in Send screen should not mess up the values

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#2244]
1. Tap Ethereum (mainnet) in Wallet tab
2. Tap Send
3. Enter 1.01 ETH
4. Tap "ETH" to toggle to USD
5. Tap "USD" to toggle back to ETH

Repeat 4 and 5 a few times, the USD will go totally wrong and if tapped again ETH = 0. It *looked* like the USD value multipled by the price, so USD is displayed as (assuming price is $370): 136900 (370x370). Maybe a race condition somewhere?

**Comment by hboon** [comment]
Need to relook at this after #2275 is merged

**Comment by hboon** [comment]
Didn't undo #2249, so this is already done.
